gen.go

```diff
@@ -14,7 +14,7 @@
 
 package trillian
 
-//go:generate protoc -I=. -I=$GOPATH/src/github.com/googleapis/googleapis --go_out=paths=source_relative:. --go-grpc_out=paths=source_relative:. --go-grpc_opt=require_unimplemented_servers=false trillian_log_api.proto trillian_admin_api.proto trillian.proto --doc_out=markdown,api.md:./docs/
+//go:generate protoc -I=. -I=third_party/googleapis --go_out=paths=source_relative:. --go-grpc_out=paths=source_relative:. --go-grpc_opt=require_unimplemented_servers=false trillian_log_api.proto trillian_admin_api.proto trillian.proto --doc_out=markdown,api.md:./docs/
```

Comment:
Is there any prescribed naming convention for this? I'm thinking between `vendor` and `third_party`? Do you have any distinction in mind?

Reply:
I think it's arbitrary, but I'm happy to change it if the strong preference goes the other way. I'm perfectly ambivalent in this moment.

Reply:
This reading might help to decide between the 2: https://github.com/golang-standards/project-layout
(haven't read it yet myself)

Reply:
After a brief glance, `third_party` seems more relevant, esp. that it's a subset of the files (i.e. a special kind of "fork"). WDYT?